sdk/healthinsights/azure-health-insights-radiologyinsights/README.md

```diff
@@ -0,0 +1,122 @@
+# Azure Cognitive Services Health Insights Radiology Insights client library for Java
+
+
+[Health Insights][health_insights] is an Azure Applied AI Service built with the Azure Cognitive Services Framework, that leverages multiple Cognitive Services, Healthcare API services and other Azure resources.
+
+[Radiology Insights][radiology_insights_docs] is a model that aims to provide quality checks as feedback on errors and inconsistencies (mismatches) and ensures critical findings are identified and communicated using the full context of the report. Follow-up recommendations and clinical findings with measurements (sizes) documented by the radiologist are also identified.
+[Product Documentation][product_documentation]
+
+## Getting started
+
+### Prerequisites
+
+- A [Java Development Kit (JDK)][jdk] with version 8 or above
+- [Azure Subscription][azure_subscription]
+- An existing Cognitive Services Health Insights instance.
+
+For more information about creating the resource or how to get the location and sku information see [here][cognitive_resource_cli].
+
+### Adding the package to your product
+
+[//]: # ({x-version-update-start;com.azure:azure-health-insights-radiologyinsights;current})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-health-insights-radiologyinsights</artifactId>
+    <version>1.0.0-beta.1</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+### Authenticate the client
+
+In order to interact with the Health Insights Radiology Insights service, you'll need to create an instance of the RadiologyInsightsClient class.  You will need an **endpoint** and an **API key** to instantiate a client object.
+
+#### Get API Key
+
+You can obtain the endpoint and API key from the resource information in the [Azure Portal][azure_portal].
+
+Alternatively, you can use the [Azure CLI][azure_cli] snippet below to get the API key from the Health Insights resource.
+
+```bash
+az cognitiveservices account keys list --resource-group <your-resource-group-name> --name <your-resource-name>
+```
+
+#### Create a RadiologyInsightsClient with an API Key Credential
+
+Once you have the value for the API key, you can pass it as a string into an instance of **AzureKeyCredential**. Use the key as the credential parameter
+to authenticate the client:
+
+```java com.azure.health.insights.radiologyinsights.buildasyncclient
+String endpoint = Configuration.getGlobalConfiguration().get("AZURE_HEALTH_INSIGHTS_ENDPOINT");
+String apiKey = Configuration.getGlobalConfiguration().get("AZURE_HEALTH_INSIGHTS_API_KEY");
+
+RadiologyInsightsAsyncClient radiologyInsightsAsyncClient = new RadiologyInsightsClientBuilder()
+        .endpoint(endpoint).serviceVersion(RadiologyInsightsServiceVersion.getLatest())
+        .credential(new AzureKeyCredential(apiKey)).buildAsyncClient();
```

Comment:
Same here, better to let user build `RadiologyInsightsClient`.

Reply:
Added example code to the README file for the synchronous client in last commit.